owner-onboarding-server/src/main.rs

```diff
@@ -346,6 +415,48 @@ async fn main() -> Result<()> {
             .context("Error converting owner public key to PK")?
     };
 
+    // Voucher registration times
+    let str_registration_period = settings.registration_period;
+    let str_re_registration_window = settings.re_registration_window;
+    let registration_period = match str_registration_period {
```

Comment:
You could do a `let registration_period = match settings.registration_period {`, which lets you get around the `let str_...`.

Reply:
Will do, good catch.